assets/blocks/button/button-props.js

```diff
@@ -60,3 +64,12 @@ export function getButtonWrapperProps( { className, attributes: { align } } ) {
 		),
 	};
 }
+
+/**
+ * Check if block has the 'Link' block style.
+ *
+ * @param {Array} props Block props.
+ * @return {boolean} Is it a link block style.
+ */
+export const isLinkStyle = ( props ) =>
+	props?.attributes?.className?.match( /\bis-style-link\b/ );
```

Comment:
We can add the `!!` to return the `boolean` as documented.

```suggestion
	!! props?.attributes?.className?.match( /\bis-style-link\b/ );
```

Reply:
Changed this to `String.prototype.test` that just checks for a match and returns a boolean. @ d74b1b6

Reply:
I think the `test` is a method from the `RegExp`. In this case, we'd need to invert the order. Or am I missing something?

Reply:
Yup, my bad, fixing in fc2d54f